kin-sdk/GeneralBlockchainInfoRetriever.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using System.Threading.Tasks;
+using kin_base;
+using kin_base.responses;
+using kin_base.responses.page;
+using kin_base.requests;
+
+namespace kin_sdk
+{
+    class GeneralBlockchainInfoRetriever
```

Comment:
I'd opt for a static class where the server context is passed to the actual methods.  Ideally server context type would be an interface to remove the coupling of the stellar sdk

Reply:
Sounds good, but maybe in the future, for now Im just copying most of the existing implementation from the android sdk